src/_main_/utils/common.py

```diff
@@ -94,6 +94,10 @@ def parse_list(d):
             return d.strip().split(",") if d else []
         elif isinstance(d, dict):
             tmp = list(d.values())
+        
+        elif isinstance(d, list):
+            if d and isinstance(d[0], dict):
```

Comment:
No need for the `tmp` variable. Also the method can be improved. Consider below:

```
def parse_list(d):
      try:
          if isinstance(d, str):
              return d.strip().split(",") if d else []
    
          if isinstance(d, dict):
              return [value for value in d.values() if str(value).isnumeric()]
    
          if isinstance(d, list):
              if d and isinstance(d[0], dict):
                  return d
              return [item for item in d if str(item).isnumeric()]
    
          return []
      except Exception as e:
          log.exception("Error in parse_list: %s", e)
          return []
```

Reply:
thanks for the suggestion.